src/bfx_tools/gatk.ml

```diff
@@ -497,6 +505,25 @@ let base_quality_score_recalibrator
       on_failure_activate (Remove.file ~run_with recal_data_table);
     ]
 
+let base_quality_score_recalibrator
+    ~configuration ~run_with ~input_bam ~result_prefix how =
+  let reference =
+    Machine.get_reference_genome run_with input_bam#product#reference_build in
+  let run_on_region region =
+    base_quality_score_recalibrator_on_region ~configuration
+      ~run_with ~input_bam ~result_prefix ~region
+  in
+  match how with
+  | `Region region -> run_on_region region
+  | `Map_reduce ->
+    let all_nodes =
+      List.map (Reference_genome.major_contigs reference)
```

Comment:
Is there an easy way for the user to specify that it should be run over all the contigs? Or would they need to enumerate the name of all the other small contigs/unmapped regions in the reference in order to get this behavior? (And thereby creating a "new" `Reference_genome.t`?)